integration_tests/projects/013_structured_hooks/fal_project.yml

```diff
@@ -9,3 +11,6 @@ environments:
   - name: pyjokes-0.6.0
     requirements:
       - pyjokes==0.6.0
+  - name: funny-conda
+    conda_requirements:
```

Comment:
What do you think of this name? Should we instead call it conda_deps? Maybe something else? (also should we now start calling `requirements` field, `pip_requirements`/`pip_deps` for the sake of consistency)

CC: @burkaygur @turbo1912 on this as well.

Reply:
> What do you think of this name? Should we instead call it conda_deps?

Follow what conda normally calls it. It seems to be packages, so we can do `conda_packages`.

> also should we now start calling requirements field, pip_requirements/pip_deps for the sake of consistency

I would agree with this. And there is no problem since this feature is not being used yet.

Reply:
Also to consider: what if we do like profiles.yml, you specify a type of env and the requirments/packages/xxx in a different property.

```yaml
  - name: funny-pip
    type: pip
    requirements:
      - pyjokes==0.6.0
  - name: funny-conda
    type: conda
    packages:
      - pyjokes==0.6.0
```

I think this may be good for a future where env types have different properties to set (ram, gpu, etc)

Reply:
I think this is even better, since it would also reduce the ambiguity on defaults (e.g. you would be able to just specify `type: fal-cloud` which would have a default machine type, instead of specifying `machine` explicitly as we are doing now).

Reply:
@chamini2 should be implemented now! I think it looks fantastic with your suggestion